Examples/Detectors/CMakeLists.txt

```
@@ -5,3 +5,5 @@ add_subdirectory_if(Geant4Detector ACTS_BUILD_EXAMPLES_GEANT4)
 add_subdirectory(MagneticField)
 add_subdirectory(TGeoDetector)
 add_subdirectory(TelescopeDetector)
+add_subdirectory(MuonSpectrometerMockupDetector)
```

```suggestion
add_subdirectory_if(MuonSpectrometerMockupDetector ACTS_BUILD_EXAMPLES_GEANT4)
```

that should fix the problem @dimitra97